lib/topological_inventory/amazon/collector.rb

```diff
@@ -1,35 +1,34 @@
 require "concurrent"
+require "topological_inventory-ingress_api-client/collector"
 require "topological_inventory/amazon/connection"
-require "topological_inventory/amazon/collector/cloud_formation"
-require "topological_inventory/amazon/collector/ec2"
-require "topological_inventory/amazon/collector/pricing"
-require "topological_inventory/amazon/collector/service_catalog"
 require "topological_inventory/amazon/parser"
 require "topological_inventory/amazon/iterator"
 require "topological_inventory/amazon/logging"
 require "topological_inventory-ingress_api-client"
-require "topological_inventory-ingress_api-client/save_inventory/saver"
 
 module TopologicalInventory
   module Amazon
-    class Collector
+    class Collector < ::TopologicalInventoryIngressApiClient::Collector
       include Logging
+      
+      require "topological_inventory/amazon/collector/cloud_formation"
```

Comment:
@Ladas it's needed to move there because it cannot load classes properly when it has superclass

Reply:
ok, lets see if we can do this nicer later

@agrare do you know a nicer solution? The issue is that if we load the mixins first, it loads the class that doesn't inherit from anything:

```
module TopologicalInventory
  module Amazon
    class Collector
      module CloudFormation
```

then loading `class Collector < ::TopologicalInventoryIngressApiClient::Collector` fails